app/config/network_configuration.py

```diff
@@ -28,35 +28,36 @@ def get_node_logs_link():
         return node_logs_link
 
 
-def network_ws_endpoint():
-    return get_var_from_env('WS_ENDPOINT')
+def relay_chain_rpc_url():
+    return get_var_from_env('RELAY_CHAIN_RPC_URL')
```

Comment:
Could you incorporate backward compatibility or enforce migration?

For instance, if a new environment variable is not present, you could check the old value. In cases where the old name is present, you have two options:

1. **Force Migration:** If you detect the old environment variable, the migration process can be enforced. In this scenario, the system would fail with an error and request the removal of the old environment variable. 

2. **Backward Compatibility:** Alternatively, you could opt for backward compatibility. If the old environment variable is found, the system would use the old value while also printing a warning. This approach allows for continued functioning with the old configuration while alerting users that the environment variable is deprecated and should be updated.